packages/cardano-services/test/cli.test.ts

```diff
@@ -22,7 +22,7 @@ import axios, { AxiosError } from 'axios';
 import http from 'http';
 import path from 'path';
 
-jest.setTimeout(60_000);
+jest.setTimeout(90_000);
```

Comment:
Was it being flaky with a 60 second timeout?

Reply:
No actually, but I noticed the average `cli.test.ts` execution time is around 60 secs, so I increased it to 1.5 min as a precaution

Reply:
It's OK here for now, but in the future it would be better to commit this separately with a description to explain the change.